RELEASE.md

```diff
@@ -0,0 +1,88 @@
+testing. Generally, it's best to look through all the pages and check if anything
+looks off.
+
+#### Script
+
+Actions in the script should be done under the `cf-deck-testing` organization
+and the `testSpace01` space.
+
+1. Go to `/` and ensure everything looks OK.
+1. Click on an organization on the right hand side nav.
+  - Ensure that the correct organization get's highlighted in the sidenav.
+  - Ensure the menu correctly opens.
+1. Click on `Marketplace` in the sidenav and ensure the page appears and looks OK.
+1. Click `Create service instance` on any of the rds service plans.
+  - Ensure you are easily able to see the `Create service instance` form.
+  - Ensure the `Create service instance` form looks OK.
+1. Choose a memorable name for the service instance and type it into the first textbox.
+1. Add the service instance under a the `testSpace01` space by selecting it from the dropdown.
+1. Click the `Cancel` button and ensure the form correctly disappears.
+1. Repeat the last few steps to open the `Create service instance` form again filled out.
+1. Click the `Create service instance` button and ensure the form disappears.
+1. Click on the `spaces` link and ensure the all spaces page appears and looks OK.
+1. Click on a space on the sidnav in the list of spaces.
+  - Ensure the space get's correctly highlighted in the sidenav.
+  - Ensure the specific space page appears and looks OK.
+1. Click on the `Service instances` link in the subnav.
+  - Ensure the page appears and looks OK.
+  - Ensure that your test service instance you created is in the list of instances.
+1. Click the `Delete instance` button for the instance you created.
+  - Ensure a delete confirmation form appears.
+1. Click the `Cancel` button on the delete confirmation form and ensure the form disappears.
+1. Click the `Delete instance` button for the instance you created again.
+1. Click 'Confirm delete' and ensure the form and the instance disappear after a few seconds.
+1. Click on `User management` in the subnav and ensure the page appears and looks OK.
+1. Attempt to click on the role checkboxes for a user. (If it's grayed out, you do not have permission to edit it, which is OK)
+  - Ensure the checkbox becomes unchecked after a second of clicking on it.
+1. Click on the same checkbox again to keep the permission the same
+  - Ensure the checkbox becomes unchecked after a second of clicking on it.
+1. Click on the 'All organization users` link in the subnav.
+1 Repeat steps to check a user permission.
+1. Techinically, we shoudl test removing a user from an org here, but will not because theres no easy way to add them back.
+1. Click on the `Apps` link in the subnav
+1. Click on `testapp01` or any app available and ensure the page appears and looks OK.
+1. Click on the `Overview` link in the sidenav and ensure the homepage appears.
+
+## Tagging
+
+Once the site looks OK and ready to deploy, tagging a commit in git will trigger
+the deploy process to production.
```

Comment:
By "production" do you mean https://dashboard-master.cloud.gov/ or https://dashboard.cloud.gov/  or something else?